backend/backend.go

```diff
@@ -555,6 +568,68 @@ func (backend *Backend) Coin(code coinpkg.Code) (coinpkg.Coin, error) {
 	return coin, nil
 }
 
+func (backend *Backend) pollETHAccounts() {
```

Comment:
When the app is started, time.After(0) will be handled immediately, likely before any accounts are loaded. When the accounts are then added (or when an ETH account is added manually later, etc), they will not be initialized for up to 5 minutes...